src/load/azext_load/data_plane/utils/models.py

```diff
@@ -11,6 +11,12 @@ class IdentityType(str, Enum):
     UserAssigned = "UserAssigned"
 
 
+class EngineIdentityType(str, Enum):
```

Comment:
I dont think we need this, this should be already present in the SDK models now?

Reply:
Can retain it for now though